### PR TITLE
Customise the meta description to describe the URL

### PR DESCRIPTION
**Pull request by str4d**
This is useful when pasting permalinks into apps that show the description in a summary of the page.

### CODE REVIEW

File: codespeed/views.py [hunk 1]
```diff
@@ -611,7 +616,10 @@ def changes(request):
         ]
     revisionlists = json.dumps(revisionlists)
 
+    pagedesc = "Report of %s performance changes for commit %s" % \
+        (defaultexecutable.name, selectedrevision.commitid)
```
Comment (tobami) [inline codespeed/views.py]: "... change for commit X on branch Y"?

File: codespeed/views.py [hunk 2]
```diff
@@ -463,11 +463,16 @@ def timeline(request):
         defaultextr = "on"
 
     # Information for template
+    if defaultbenchmark in ['grid', 'show_none']:
+        pagedesc = None
+    else:
+        pagedesc = "Results timeline for the '%s' benchmark" % defaultbenchmark.name
```
Comment (tobami) [inline codespeed/views.py]: "... for the X benchmark (project Y)"?

**Review by tobami — CHANGES_REQUESTED** [review]
Looks grand. Just see whether the descriptions themselves can be a bit more specific considering environments, branches and projects.

**Comment by str4d** [comment]
Force-pushed to squash in review changes, and also switch to using the `__str__()` representation for consistency with other parts of the UI.

**Review by tobami — APPROVED** [review]

**Pull request merged**